core/state_transition.go

```diff
@@ -335,7 +335,7 @@ func (st *StateTransition) TransitionDb() (*ExecutionResult, error) {
 		ret, st.gas, vmerr = st.evm.Call(sender, st.to(), st.data, st.gas, st.value)
 	}
 
-	if !rules.IsLondon {
+	if !rules.IsLondon && !rules.IsMystique {
```

Comment:
@meowsbits, do you think that `IsETCMystique` is a better name, so as devs can see that it's similar forks and not two different forks on the same chain?

Reply:
This idea applies to all our forks